sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -420,29 +466,6 @@ def __init__(self, kind) -> None:
 ]
 
 
-@pytest.fixture
```

Comment:
Do you need to move this fixture before we define the helper functions ?
I think, the location where this fixture sits looks good to me (e.g. L423).

Reply:
Moved that for my own convenience while fixing the get_job unit test after rebase. I will move it back :+1: